.github/workflows/release.yml

```diff
@@ -131,11 +120,22 @@ jobs:
         prerelease: false
 
     - name: Publish to Pypi
-      run: echo Publishing to Pypi
+      uses: pypa/gh-action-pypi-publish@v1.0.0a0
+      with:
+        user: firebase
+        password: ${{ secrets.FIREBASE_PYPI_PASSWORD }}
 
     # Post to Twitter if explicitly opted-in by adding the label 'release:tweet'.
     - name: Post to Twitter
       if: success() &&
         contains(github.event.pull_request.labels.*.name, 'release:tweet')
-      run: echo Posting Tweet
+      uses: ethomson/send-tweet-action@v1
+      with:
+        status: >
+          ${{ steps.preflight.outputs.version }} of @Firebase Admin Python SDK is avaialble.
+          https://github.com/firebase/firebase-admin-python/releases/tag/${{ steps.preflight.outputs.version }}
+        consumer-key: ${{ secrets.FIREBASE_TWITTER_CONSUMER_KEY }}
+        consumer-secret: ${{ secrets.FIREBASE_TWITTER_CONSUMER_SECRET }}
```

Comment:
Not critical, but I was wondering whether we should remove the `FIREBASE_` prefix from these secrets.
`TWITTER_CONSUMER_SECRET` instead `FIREBASE_TWITTER_CONSUMER_SECRET`

Reply:
Done